servo/src/server/routes/mappings.rs

```diff
@@ -18,6 +16,261 @@ pub struct GetMappingResponse {
   pub mappings: Vec<NodeMapping>,
 }
 
+const PYTHON_KEYWORDS : [&str; 70] = [
```

Comment:
Solid, this looks like a good list. I'm surprised Rust doesn't require you to specify the lifetime, as in `[&'static str, 70]`, but if this compiles then it's good.